Source/Particles/PhysicalParticleContainer.H

```diff
@@ -72,9 +72,9 @@ public:
     void InitIonizationModule () override;
 
     /*
-     * \brief Returns a pointer to the plasma injector.
+     * \brief Returns a reference to the i'th plasma injector.
```

Comment:
It seems that it is still a pointer.
```suggestion
     * \brief Returns a pointer to the i'th plasma injector.
```